internals/reminders.py

```diff
@@ -57,25 +57,27 @@ def get_current_milestone_info(anchor_channel: str):
 def choose_email_recipients(
     feature: FeatureEntry, is_escalated: bool, is_accuracy_email: bool) -> list[str]:
   """Choose which recipients will receive the email notification."""
+  ws_group_emails = []
+  if is_escalated:
```

Comment:
I think we want to include these mailing lists regardless of is_escalated. Johnny was checking on how many regular notifications went out each week, not just escalations.